twake/backend/node/src/services/files/entities/file.ts

```diff
@@ -0,0 +1,29 @@
+import { Type } from "class-transformer";
+import { Entity, Column } from "../../../core/platform/services/database/services/orm/decorators";
+
+@Entity("files", {
+  primaryKey: [["name"], "id"],
+  type: "channels",
+})
+export class File {
+  // uuid-v4
+
+  @Type(() => String)
+  @Column("id", "uuid", { generator: "uuid" })
+  id: string;
+
+  @Column("name", "encoded_string")
+  name: string;
+
+  @Column("size", "encoded_string")
+  size: number;
+
+  @Column("width", "encoded_string")
+  width: number;
+
+  @Column("height", "encoded_string")
+  height: number;
```

Comment:
These are parameters which will be used for images only, right? What about putting them in a JSON metadata field?